docs/modules/password_auth_provider_callbacks.md

```diff
@@ -103,6 +103,9 @@ Called during a logout request for a user. It is passed the qualified user ID, t
 deactivated device (if any: access tokens are occasionally created without an associated
 device ID), and the (now deactivated) access token.
 
+Deleting the related pushers is done after calling `on_logged_out`, so you can rely on them
+to still be present.
```

Comment:
Can we add a test so we don't break it again?

Reply:
Good point, will do.